arch/bit_range_ops.h

```diff
@@ -1,18 +1,18 @@
-// This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
-
 /* bit_range_ops.h                                                 -*- C++ -*-
    Jeremy Barnes, 23 March 2009
    Copyright (c) 2009 Jeremy Barnes.  All rights reserved.
 
+   This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
```

Comment:
The extra header stopped emacs from being able to detect C++ files, which caused indenting issues, so I fixed the headers in the files I edited.  We should probably do a round of improvements to the header messages.